snap/snapcraft.yaml

```diff
@@ -20,13 +20,13 @@ adopt-info: certbot
 
 apps:
   certbot:
-    command: certbot.wrapper
+    command: bin/python3 -m certbot
```

Comment:
Something is going wrong with logging and error handling because of this change:

    $ sudo -E ./venv3/bin/certbot --apache
    Saving debug log to /var/log/letsencrypt/letsencrypt.log
    The apache plugin is not working; there may be problems with your existing configuration.
    The error was: NoInstallationError('Cannot find Apache executable apache2ctl')

  versus what happens in the snap right now:

    $ sudo -E ./venv3/bin/python3 -m certbot --apache
    Saving debug log to /var/log/letsencrypt/letsencrypt.log

If I understand right, you made this change because of Debian 9's dislike of `$SNAP`.

However, as you correctly pointed out, we can't really do anything about Debian 9 anyway, shall we revert this to what it previously was?

Reply:
Ok. I reverted to the `bin/python3 $SNAP/bin/certbot` approach. I also double-checked that, on top of having the proper error handling on snap connection failing, the `/snap/bin/certbot --apache` call is failing also with the proper error.